addons/remember-collapsed-categories/addon.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "Remember Closed Categories",
+  "description": "Fixes the forums not remembering what forum categories you've closed.",
```

May I suggest a new name and description?
```suggestion
  "name": "Remember collapsed forum categories",
  "description": "Categories you collapsed on the forum home page (the list of forums) will stay collapsed.",
```

Why my description change? To most people, this will seem like a new feature rather than a fix.

---

I don't think we should try really hard to make it clear this is technically a bug fix. Most users aren't aware of this broken feature.